web/src/main/java/com/microsoft/applicationinsights/web/internal/correlation/tracecontext/Tracestate.java

```diff
@@ -0,0 +1,50 @@
+package com.microsoft.applicationinsights.web.internal.correlation.tracecontext;
+
+import org.apache.http.annotation.Experimental;
+
+/**
+ * Class that represents Tracestate header based on
+ * @link https://github.com/w3c/trace-context/blob/master/trace_context/HTTP_HEADER_FORMAT.md
+ *
+ * According to W3C specification, implementations should pass the tracestate header to downstream
+ * calls if received.
+ *
+ * Implementations can add vendor specific details here.
+ *
+ * @author Reiley Yang
```

Comment:
Curious, why do we put author name here?

Reply:
Last time I've seen this was when I work on the C++ compiler, at that time there was no "git blame".

Reply:
Most Java libraries do this :) Its a common pattern in Java projects.